app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="baseTextStyle" parent="TextAppearance.AppCompat.Display1">
```

Comment:
pls each style need to be written in UpperCase

Reply:
done